repositories/src/main/scala/cromwell/pipeline/datastorage/dto/WomToolNode.scala

```diff
@@ -0,0 +1,43 @@
+package cromwell.pipeline.datastorage.dto
+
+import play.api.libs.json._
+
+final case class WomToolNode(name: String, value: Option[String])
+
+object WomToolNode {
+  implicit object WomToolNodeFormat extends Format[WomToolNode] {
+    override def writes(o: WomToolNode): JsValue = JsObject(Seq(o.name -> JsString(optToString(o.value))))
```

Comment:
Is there any reason why you prefer creating JsObject manually instead of type-safe `json` combinators?
https://www.playframework.com/documentation/2.8.x/ScalaJsonCombinators

Reply:
Yes, because we must pass nodes to wdl like this: `node_name -> node_value`. I didn't find any information of how to do like so without JsObject

Reply:
I've thought you'll do this differently :)
Anyway, this is a strange choice of serialization approach.
Suppose you have the following
`val node = WomToolNode("thisIsName", Some("thisIsValue"))`
And you are trying to serialize it as follows
```
{
  "thisIsName" : "thisIsValue"
}
```
instead of
```
{
  "name" : "thisIsName",
  "value" : "thisIsValue"
}
```
Why are you doing it? Is there any particular reason or this is a mistake?

Reply:
That's because cromwell take input params like so